doc/shield-coinbase.md

```diff
@@ -0,0 +1,101 @@
+# Shielding Coinbase UTXOs
+
+**Summary**
+
+Use z_shieldcoinbase RPC call to shield coinbase utxos.
+
+**Who should read this document**
+
+Miners, Mining pools, Online wallets
+
+## Background
+
+The current Zcash protocol includes a consensus rule that coinbase rewards must be sent to a shielded address.
+
+## User Experience Challenges
+
+A user can use the z_sendmany RPC call to shield coinbase funds, but the call was not designed for sweeping up many utxos, and offered a suboptimal user experience.
+
+If customers send mining pool payouts to their online wallet, the service provider must sort through UTXOs to to correctly determine the non-coinbase UTXO funds that can be withdrawn or transferred by customers to another transparent address.
+
+## Solution
+
+The z_shieldcoinbase call makes it easy to sweep up coinbase rewards from multiple coinbase utxos across multiple coinbase reward addresses.
+
+    z_shieldcoinbase fromaddress toaddress (fee) (limit)
+
+Where the default fee is 0.0010000 ZEC and the limit is 50.
```

Comment:
"The default fee is 0.0010000 ZEC and the default limit on the maximum number of UTXOs to shield is 50."

Reply:
ok